modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+// CNF-950 4.7 Provisioning and deploying a Distributed Unit (DU) manually
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
```

Comment:
* scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc

Reply:
Approved.